doc/JSON_FLAGS.md

```diff
@@ -944,8 +944,11 @@ Those flags are added by the game code to specific items (that specific welder,
 - ```FISHING_BASIC``` Use a fishing rod
 - ```JET_INJECTOR``` Inject some jet drugs right into your veins.
 - ```CABLE_ATTACH``` This item is a cable spool. Use it to try to attach to a vehicle.
-
-
+- ```CAPTURE_MONSTER_ACT``` Capture and encapsulate a monster. This action is affected by several flags which as of this point, have no other purposes. The associated action is also used for releasing it.
+  - ```NO_BREAK_ON_FAIL``` Prevents a monster capturing device, like a pet carrier, from breaking if it fails to capture the monster.
```

Comment:
How is this flag supposed to affect anything when the only place it's ever referenced is in the item's entry and this documentation file? If it's to stop *foo* from breaking, then it needs to be checked for in the code that handles breaking *foo*.